netsim/providers/clab.yml

```diff
@@ -26,6 +28,8 @@ kmods:
   vxlan: [ vxlan, udp_tunnel, ip6_udp_tunnel ]
   vrf: [ vrf ]
 attributes:
+  global:
```

Comment:
Why are you defining global attribute if you read the value from the system defaults?

Reply:
I was thinking it would be nice to have type validation including string length limitation, but I didn't finish my thought. Will rework

Reply:
We don't do type validation for default settings

Reply:
Hence my thought to make it a global attribute instead

Reply:
Then you should change all references to it

Reply:
It turned out to be more complicated to do a global attribute, and none of the providers currently have any. Decided to stick with the defaults section

Added a paragraph to documentation